### ISSUE TITLE
powershell install on Kali 2019.4

### ISSUE THREAD

**Issue by dpareit** [MicrosoftDocs/PowerShell-Docs#5206]
Hi, 
Note that powershell installation on Kali 2019.4 no longer requires extra apt sources/key.
It's now as easy as: "apt install -y powershell"
Cfr https://www.kali.org/news/kali-linux-2019-4-release/

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 37f30869-1dcc-9b77-ecf3-e0ce88bce104
* Version Independent ID: a6f4b56d-aa6c-b557-c0d2-1ff6888c24bd
* Content: [Installing PowerShell Core on Linux - PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-linux?view=powershell-6)
* Content Source: [reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/docs-conceptual/install/Installing-PowerShell-Core-on-Linux.md)
* Product: **powershell**
* Technology: **powershell-conceptual**
* GitHub Login: @joeyaiello
* Microsoft Alias: **jaiello**

**Comment by sdwheeler** [comment]
@TravisEz13 Can you comment/confirm?

**Comment by tntna** [comment]
Im using kali on the RPI3. And its just not happening. Literally over a full day spent trying. This is what i get all the time.

Package powershell is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
E: Package 'powershell' has no installation candidate

Following the kali website word for word. Its only 1 step anyways. Cant really get it that wrong.

Tried every single instruction on this entire page. --> https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-linux?view=powershell-6 plus many many more.

I got the snap file working in a seperate ubuntu installation. But majority of the commands in powershell result in this same error over and over. Even the Get-Help and Update-Help

Update-Help : Read-only file system
At line:1 char:1
+ Update-Help
+ ~~~~~~~~~~~
+ CategoryInfo          : NotSpecified: (:) [Update-Help], IOException
+ FullyQualifiedErrorId : System.IO.IOException,Microsoft.PowerShell.Commands.UpdateHelpCommand

:-(   why....
